app/models/concerns/labware_creators/require_wells_with_collected_by.rb

```diff
@@ -8,7 +8,7 @@ module LabwareCreators::RequireWellsWithCollectedBy
   # have an associated sample metadata, with collected_by.
   def wells_with_aliquots_have_collected_by?
     invalid_well_locations = wells_with_missing_collected_by
-    return if invalid_well_locations.empty?
+    return false if invalid_well_locations.empty?
```

Comment:
this will cause a conflict with Andrew's branch, but easy to resolve hopefully as he's seen this already